moveit_ros/visualization/motion_planning_rviz_plugin/src/motion_planning_frame_objects.cpp

```diff
@@ -215,22 +212,29 @@ void MotionPlanningFrame::sceneScaleEndChange()
 
 void MotionPlanningFrame::removeSceneObject()
 {
-  QList<QListWidgetItem*> sel = ui_->collision_objects_list->selectedItems();
-  if (sel.empty())
+  QList<QListWidgetItem*> selection = ui_->collision_objects_list->selectedItems();
+  if (selection.empty())
     return;
-  planning_scene_monitor::LockedPlanningSceneRW ps = planning_display_->getPlanningSceneRW();
-  if (ps)
+
+  if (planning_scene_monitor::LockedPlanningSceneRW ps = planning_display_->getPlanningSceneRW())
```

Comment:
I don't think this `if()` does anything. It looks like [`getPlanningSceneRW()`](https://github.com/ros-planning/moveit/blob/1f4435190f7498d746b8066fb72c956eebe8eeef/moveit_ros/visualization/planning_scene_rviz_plugin/src/planning_scene_display.cpp#L300) and the [constructor of LockedPlanningSceneRW](https://github.com/ros-planning/moveit/blob/1f4435190f7498d746b8066fb72c956eebe8eeef/moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h#L614) cannot fail.

Reply:
`planning_scene_monitor_` could be a `nullptr`, couldn't it?
https://github.com/ros-planning/moveit/blob/1f4435190f7498d746b8066fb72c956eebe8eeef/moveit_ros/planning/planning_scene_monitor/include/moveit/planning_scene_monitor/planning_scene_monitor.h#L625-L628